pkg/stanza/fileconsumer/attrs/ownerInfo_linux.go

```diff
@@ -0,0 +1,34 @@
+//go:build linux || solaris
```

Comment:
I tested this and it works on darwin as well. Any reason to believe it fails outside of windows?

```suggestion
//go:build !windows
```

We should also change the name to snake case, maybe just `owner_other.go`.